tensorrt_llm/serve/openai_video_routes.py

```
@@ -0,0 +1,440 @@
+            media_type = "video/mp4" if actual_path.suffix == ".mp4" else "video/x-msvideo"
+
+            return FileResponse(
+                actual_output_path,
+                media_type=media_type,
+                filename=actual_path.name,
+            )
+
+        except ValueError as e:
+            logger.error(f"Request parsing error: {e}")
+            return self.create_error_response(str(e))
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)